source/usage-examples/changeStream.txt

```diff
@@ -88,17 +88,17 @@ print change events that occur on the collection.
 
 First, open the change stream on the collection and then define a callback
 on the change stream using the ``on()`` method. Once set, generate a change
-event to be emitted by performing a change to the collection.
+event to emit by performing a change to the collection.
 
 To generate the change event on the collection, let's use ``insertOne()``
 method to add a new document. Since the ``insertOne()`` may run before the
-listener function can register, we use a timer, declared with ``setTimeout()``
-to wait one second before executing the insert.
+listener function can register, we use a timer, declared with
+``simulateAsyncPause`` to wait 100 milliseconds before executing the insert.
```

Comment:
```suggestion
listener function can register, we use a timer to wait 100 milliseconds before executing the insert.
```
Suggestion: I'm not sure "declared with" is right here. I think you could say "defined as", or I think it could just be called a timer.